database/src/main/kotlin/no/nav/su/se/bakover/database/søknad/SøknadRepoInternal.kt

```diff
@@ -25,8 +25,12 @@ internal object SøknadRepoInternal {
 
 internal fun Row.toSøknad(): Søknad {
     return Søknad(
+        sakId = uuid("sakId"),
         id = uuid("id"),
         søknadInnhold = objectMapper.readValue(string("søknadInnhold")),
-        opprettet = tidspunkt("opprettet")
+        opprettet = tidspunkt("opprettet"),
+        lukket = stringOrNull("lukket")?.let {
+            objectMapper.readValue<LukketSøknadJson>(it).toLukket()
```

Comment:
Vi ser for oss at domene-typen Søknad kommer til å forandre seg drastisk, og tar litt høyde for det her ved å ha et lag i mellom.